tls/s2n_early_data.c

```diff
@@ -339,7 +339,20 @@ int s2n_connection_get_max_early_data_size(struct s2n_connection *conn, uint32_t
     POSIX_ENSURE_REF(max_early_data_size);
     *max_early_data_size = 0;
 
+    uint32_t server_max_early_data_size = 0;
+    POSIX_GUARD_RESULT(s2n_early_data_get_server_max_size(conn, &server_max_early_data_size));
+
     if (conn->psk_params.psk_list.len == 0) {
+        /* The server can load its PSKs during the handshake, either via the PSK selection callback
+         * or by receiving a stateless session ticket.
```

Comment:
Nit: I would be more specific and call out PSKs being loaded after this API call: 

`The server can load its PSKs after calling this API during the handshake`